muesli/web/static/js/accidental-action-prevention.js

```diff
@@ -0,0 +1,23 @@
+function onActionClicked(event) {
```

Comment:
Rename this to accidentialClickPrevention

Reply:
Changed.